src/main/scala/leon/purescala/MethodLifting.scala

```diff
@@ -287,7 +288,7 @@ object MethodLifting extends TransformationPhase {
     // 5) Replace method calls with function calls
     for (fd <- pgm.definedFunctions) {
       fd.fullBody = postMap{
-        case mi @ MethodInvocation(IsTyped(rec, ct: ClassType), cd, tfd, args) =>
+        case mi @ MethodInvocation(IsTyped(rec, ct: ClassType), cd, tfd, args) if !tfd.fd.isLibrary =>
```

Comment:
@ravimad Now with the new command line option integration with leon will be easy but this line needs updating.
We should be doing something line `if (!tfd.fd.isLibrary || !printRunnableCode)` but I don't know how to get the PrinterContext here (or how to get it from LeonContext).
I'd like to fix this before I merge this PR